.github/workflows/code_style.yml

```
@@ -21,11 +21,12 @@ jobs:
             git
 
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Fetch ${{ github.base_ref }}
         shell: bash
         run: |
+          git config --global --add safe.directory `pwd`
```

```suggestion
          git config --global --add safe.directory $(pwd)
```
Better to prefer `$()` over backticks. https://mywiki.wooledge.org/BashFAQ/082

---

I will fix this.

---

Done.

---

Sorry, I meant the other ones as well :sweat_smile:. Also, please use `[ci]` instead of `[github-workflows]`.

---

Done.